CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added method to AbstractGridFactory(), `get_subset()` that is used by time dependent grids to return the corners of the  subset of the grid that is active during the current time interval.
+- Added ability to delete regridders and grid factories including proper destruction of associated ESMF routehandles and grids.
+- Added support for use of pFlogger simTime in logging (only if `-DBUILD_WITH_PFLOGGER=ON`)
+  - Note: Due to bug in pFlogger v1.9.3 and older, you *must* specify a `dateFmt` in your logging configuration file in the
+    formatter when using `simTime` (see [pFlogger issue #90](https://github.com/Goddard-Fortran-Ecosystem/pFlogger/issues/90))
+- Add geom subdirectory and contents for MAPL Geom framework
+
```

Comment:
Something is wrong.   You changes should not be bringing in all of these  other items.   Usually this means that you started your branch from main instead of develop, but it could also be something else.

@mathomp4 may be able to help you correct this next week. (He's off today.)

Reply:
Oh, I see.  It was because when I was using your branch to support delete grid and RH, I merged your code along with the changelog. I will delete this. 
```
Merge remote-tracking branch 'origin/feature/tclune/#2086-delete-regridder' into feature/ygyu/epoch_swatch_v0.0
```

Reply:
I have deleted the inaccurate descriptions containing so much details.  It will be replaced by a single item: Add Epoch SwathGrid for swath sampler